mifosng-android/src/main/java/com/mifos/mifosxdroid/online/search/SearchFragment.java

```diff
@@ -124,6 +118,12 @@ public void onClickSearch() {
 
     @Override
     public void showSearchedResources(List<SearchedEntity> searchedEntities) {
+        layoutManager = new LinearLayoutManager(getActivity());
+        layoutManager.setOrientation(LinearLayoutManager.VERTICAL);
+        rv_search.setLayoutManager(layoutManager);
+        rv_search.addOnItemTouchListener(new RecyclerItemClickListener(getActivity(), this));
+        rv_search.setHasFixedSize(true);
+        rv_search.setAdapter(searchAdapter);
         searchAdapter.setSearchResults(searchedEntities);
```

Comment:
Please revert all these changes, it is not a good practice from memory optimization and memory allocation perspective because creating layout manager object,  setting itmeTouchListener, size, and adapter to search view every time when we make search is a heavier task and needs memory allocation for each that is why the previous place is better, there, it needs to be initialized once during the creation of fragment only.